pylabrobot/plate_reading/plate_reader.py

```diff
@@ -48,12 +48,16 @@ def __init__(
 
   def assign_child_resource(self, resource: Resource, location: Optional[Coordinate]=None,
                             reassign: bool = True):
-    location = location or get_child_location(resource)
     if len(self.children) >= 1:
       raise ValueError("There already is a plate in the plate reader.")
     if not isinstance(resource, Plate):
       raise ValueError("The resource must be a Plate.")
-    super().assign_child_resource(resource, location=location)
+
+    if location is not None:
+      # Directly call Machine's assign_child_resource method to allow for custom location
+      Machine.assign_child_resource(self, resource, location=location, reassign=reassign)
+    else:
+      super().assign_child_resource(resource, reassign=reassign)
```

Comment:
this mimics the previous behavior, but not sure that is truly the behavior we desire.

Reply:
I think we should implement this check in the mixin: it shouldn't update the location if one is explicitly passed